src/pymor/algorithms/rand_la.py

```diff
@@ -153,24 +172,19 @@ def find_range(self, basis_size=None, tol=None):
             if basis_size is not None:
                 block_size = min(block_size, basis_size - len(Q[-1]))
 
-            V = A.source.random(block_size, distribution='normal')
-            if self.iscomplex:
-                V += 1j*A.source.random(block_size, distribution='normal')
+            V = self._draw_samples(block_size)
+            self.Omega.append(V)
```

Comment:
```suggestion
            self.Omega.append(V, remove_from_other=True)
```
(avoid copy/free memory)